site/src/pages/components/invokers.explainer.mdx

```diff
@@ -490,11 +451,16 @@ open and present on the page and the invoking button is still navigable. A
 modal dialog renders the rest of the page inert meaning that it won't be
 possible to navigate to the invoking button.
 
+##### aria-controls
+
+Similar to `aria-details`, `aria-controls` is not useful for a dialog that
+is closed, and non-navigable for a dialog that is open as modal, so it would be
+useless to add.
```

Comment:
wording nit to not be redundant in saying it 'is not useful' and then 'it would be useless...."

```suggestion
is closed, and non-navigable for a dialog that is open as modal, so it would 
provide no benefit to add.
```